tools/test.py

```diff
@@ -1485,8 +1483,8 @@ def ExpandCommand(args):
     return ExpandCommand
   else:
     pos = value.find('@')
-    prefix = urllib_unquote(value[:pos]).split()
-    suffix = urllib_unquote(value[pos+1:]).split()
+    prefix = unquote(value[:pos]).split()
+    suffix = unquote(value[pos+1:]).split()
```

Comment:
__optional__: but in gereral, [__str.partition()__](https://docs.python.org/3/library/stdtypes.html#str.partition) is under-used and has readability advantages when you want the content both before and after the search term ('@'):
```
prefix, at, suffix = value.partition('@')
prefix = unquote(prefix).split()
suffix = unquote(suffix).split()
```

Reply:
Yup, `str.partition` is the best fit here. I updated the PR now.